run_route_scripts/create_path_pbf.sh

```diff
@@ -29,12 +31,9 @@ if [ ! -f "${CONF}" ]; then
 	usage
 fi
 
-# Set concurrency to one
-CONCURRENCY=1
-
 # Set env to save the path pbf and run the specified route
 export SAVE_PATH_PBF=true
-PATH=../build/:$PATH ./batch.sh ${INPUT} ${CONF} ${CONCURRENCY}
+cat ${INPUT} | xargs -I {} ../build/valhalla_run_route {} --config ${CONF}
```

Comment:
No need to use batch.sh to run a single route, so I updated to run valhalla_run_route directly.